src/hash_test.c

```diff
@@ -9,41 +9,41 @@
 
 
 void testcase_chunks_blake2b_512(
-        const char* chunks[],
+        const char* chunksHex[],
         uint32_t chunksCount,
-        const char* expected_
+        const char* expectedHex
 )
 {
 	PRINTF("testcase_chunks_blake2b_512\n");
 	blake2b_512_context_t ctx;
 	blake2b_512_init(&ctx);
-	uint8_t output[64];
+	uint8_t outputBuffer[64];
 	for (unsigned i = 0; i < chunksCount; i++) {
 		uint8_t chunkData[20];
```

Comment:
this one doesn't have `Buffer` postfix. I am not sure if it is worth adding it though ...